pkgs/development/python-modules/ipykernel/default.nix

```
@@ -23,6 +23,8 @@ buildPythonPackage rec {
   checkInputs = [ pytest nose ];
   propagatedBuildInputs = [ ipython jupyter_client traitlets tornado ];
 
+  patches = [ ./ipykernel_writable_temp.patch ];
```

```suggestion
  # https://github.com/ipython/ipykernel/pull/377
  patches = [ 
    (fetchpatch { 
      url = "https://github.com/ipython/ipykernel/commit/a3bf849dbd368a1826deb9dfc94c2bd3e5ed04fe.patch"; 
      sha256 = "...";
    })
  ];
```

+ adding `fetchpatch` to the header.